plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways/payment-gateways.tsx

```diff
@@ -76,6 +84,7 @@ export const PaymentGateways = ( {
 						}
 						options={ countryOptions }
 						onChange={ ( value: string ) => {
+							// save selected country and refresh the store by invalidating getPaymentProviders.
```

Comment:
Start with uppercase.

Reply:
thanks! addressed in https://github.com/woocommerce/woocommerce/pull/54263/commits/1cb4f8eb9290355d38f8cefebb9784d70bdf6450